cmd/cloud-network-config-controller/main.go

```diff
@@ -142,6 +144,28 @@ func main() {
 					}
 				}()
 				wg.Add(1)
+
+				// AWS uses a configmap "kube-cloud-config" to keep track of additional
```

Comment:
Both need it (actually I added it because I'm pretty sure it should be there in AWS, too, and I need it for OSP .. just in case that CM is updated if someone changes that certificate). I'm updating the comment and the if condition in the OpenStack PR